test-addon/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env node */
+
```

Comment:
This file should be removed and the linting rule additions should be applied to test-addon/src/.eslintrc.js (currently there are redundant .eslintrc.js in test-addon)

Reply:
(sgtm, removing.)